modules/dxp/apps/commerce/commerce-dashboard-web/src/main/java/com/liferay/commerce/dashboard/web/internal/portlet/action/CommerceDashboardForecastConfigurationAction.java

```diff
@@ -74,9 +77,15 @@ public void processAction(
 			ActionResponse actionResponse)
 		throws Exception {
 
+		ServiceContext serviceContext = ServiceContextFactory.getInstance(
+			actionRequest);
+
+		String[] assetCategoryIds = ArrayUtil.toStringArray(
+			serviceContext.getAssetCategoryIds());
+
 		setPreference(
 			actionRequest, "assetCategoryIds",
-			ParamUtil.getString(actionRequest, "assetCategoryIds"));
+			StringUtil.merge(assetCategoryIds, ","));
```

Comment:
@riccardo-ferrari We can just call this one, since it will set the default delimiter as `StringPool.COMMA`
```suggestion
			StringUtil.merge(assetCategoryIds));
```